### DISCUSSION TITLE
Upload estimation references

### DISCUSSION THREAD

**Discussion by ShadowZone** [Zuehlke/poinz#261] [Ideas]
Hi there,
I just stumbled upon this really cool tool and it's perfect for my needs! Thanks for the effort!

My team and I have a list of estimation references, just previous stories that we think are a good baseline for future story estimations, which we update regularly.

Right now we use a Mural whiteboard to track them, but in the planning session, we hop between the planning board, the external planning poker app, and the estimation reference board. That is really confusing!

A really nice feature would be if the Moderator could upload a CSV or JSON file, packed with these reference stories, and if the users click on a card, they get shown stories, which were estimated with the same value previously.

Long story short, there is a Jira planning poker plugin with screenshots. Maybe you can get there some inspiration.
![image](https://user-images.githubusercontent.com/976282/161224274-78ccf0a2-aeb7-4090-8bed-b13b07fe3d5e.png)
https://marketplace.atlassian.com/apps/1212495/planning-poker

Best Regards

**Comment by xeronimus** [comment]
HI @ShadowZone ,

Thanks for your inputs. Let's figure it out together... :)

There seem to be two feature ideas involved here

### A Import reference stories
We have a ["import csv"](https://github.com/Zuehlke/poinz/blob/master/docu/manual.md#importing-stories)  feature already in place. But I think currently we only import title + description (e.g. from Jira). 
We could extend it to also import an additional property "already present estimation"  (aka "consensus").



### B Display stories with the same value...

- The [Matrix](https://github.com/Zuehlke/poinz/blob/master/docu/manual.md#estimation-matrix) displays all estimated stories as a reference. Maybe this already serves your needs?
- ** if the users click on a card, they get shown stories, ** Would you display these "similar" stories after giving an estimate? if "auto-reveal" is on (by default), the card gets revealed immediately after everyone estimated.  so you might not see the "similar stories" display..

**Reply by xeronimus** [reply]
I just checked. we can easily extend the CSV import to also set a consensus value. We just have to think about how the column in the csv should be named. (can be multiple names, that will then be machted to "consensus").   Maybe:  "points"   "estimation"   "consensus"  "storypoints"   ?

**Reply by xeronimus** [reply]
https://github.com/Zuehlke/poinz/commit/71d412bae6f8756a06067cdd9ba4d4bfb365c394

**Reply by xeronimus** [reply]
@ShadowZone  any input?
